.github/workflows/update-perf-test-buckets.yml

```diff
@@ -42,7 +42,8 @@ jobs:
         uses: peter-evans/create-pull-request@v7
         with:
```

Comment:
❌ There are more options missing that should be aligned with other uses of this action:
```yaml
          signoff: true
          sign-commits: true
          delete-branch: true
          branch-suffix: timestamp
          labels: |
            in:building-gradle
            @dev-productivity
```